conduct.md

```diff
@@ -120,15 +120,15 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Openness. (vulnerability)
+- Retain team members ideas is a form of respecting everyones thoughts and opinions.
```

Comment:
I think rephrasing this sentence can increase its clarity. Some restructuring would be very helpful

Reply:
I like the idea, perhaps you could further highlight how students should work to prevent conflicts.